src/tagging/components/InnerComponents/TagSelector.js

```diff
@@ -55,6 +55,7 @@ class TagSelector extends React.Component {
         id="tag_cat"
         // clearable={false}
         simpleValue={false}
+        searchable={true}
```

Comment:
Linter is failing here, you can just have `searchable` here without the `={true}` part.

Reply:
Ahh, great, I assumed that, but I was not sure. Thank you! :+1: